node/plugin/tests/plugin_test.go

```diff
@@ -50,34 +66,48 @@ func TestMain(m *testing.M) {
 
 func setupAndRun(m *testing.M) {
 	ctx := context.Background()
-	rootPath := "../../../"
-
-	if testName == "" {
-		var err error
-		testName, err = deploy.CreateNewTestDirectory(templateName, rootPath)
-		if err != nil {
-			logger.Fatal("Failed to create test directory:", err)
-		}
-	}
-
-	testConfig = deploy.NewTestConfig(testName, rootPath)
-	testConfig.Deployers[0].DeploySubgraphs = false
 
-	logger.Info("Starting anvil")
 	var err error
 	anvilContainer, err = testbed.NewAnvilContainerWithOptions(ctx, testbed.AnvilOptions{
-		ExposeHostPort: true, // This will bind container port 8545 to host port 8545
```

Comment:
why remove this comment? is it no longer true?

Reply:
d9783ff0c8a26b11f37b73d1a1e981dec018fc2f